index.bs

```diff
@@ -582,37 +582,34 @@ interface ML {
 };
 </script>
 
+### Permissions Policy Integration ### {#permissions-policy-integration}
+
+This specification defines a <a>policy-controlled feature</a> identified by the
+string "<code><dfn data-lt="webnn-feature">webnn</dfn></code>".
+Its <a>default allowlist</a> is <code>'self'</code>.
+
+### The {{ML/createContext()}} method ### {#api-ml-createcontext}
 The {{ML/createContext()}} method steps are:
-1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, then throw a "{{SecurityError!!exception}}" {{DOMException}} and abort these steps.
-1. Let |promise| be [=a new promise=].
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, return [=a new promise=] [=rejected=] with a "{{SecurityError}}" and abort these steps.
+1. Return [=a new promise=] |promise| and run the following steps [=in parallel=].
 1. Let |context| be a new {{MLContext}} object.
-1. Switch on the method's first argument:
+1. Let |options| be the first argument.
+1. Switch on |option|:
     <dl class=switch>
-    <dt>{{MLContextOptions}}
+    <dt>{{MLContextOptions}} or {{undefined}}
```

Comment:
Good that you brought up the undefined type.

Should we handle undefined type as a separate case in this switch statement? Or maybe better to handle it as part of an Otherwise case to have a fall back to default for all unknown arguments including undefined?

Switch on _options_:
-> MLContextOptions
-> GPUDevice
-> Otherwise

Switch statement semantics are not specified in Web IDL, but it seems Otherwise is being popularized (out of default, otherwise, or else), notably by HTML LS.

Reply:
Let's discuss this on the call. I am open - just expressed the JS convention, meaning since we have a default value, and it falls under the same switch option, I handled them together. I am completely open on it, though.